### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by samj1912**

### CODE REVIEW

File: picard/ui/ui_infodialog.py [hunk 1]
```diff
@@ -16,7 +16,6 @@ def setupUi(self, InfoDialog):
         self.info_tab = QtWidgets.QWidget()
         self.info_tab.setObjectName("info_tab")
         self.vboxlayout = QtWidgets.QVBoxLayout(self.info_tab)
-        self.vboxlayout.setContentsMargins(0, 0, 0, 0)
```
Comment (mineo) [inline picard/ui/ui_infodialog.py]: This is not an import.

Comment (samj1912) [inline picard/ui/ui_infodialog.py]: I regenerated the UI files from .ui files. Guess that isn't needed anymore and wasn't updated in the repo. Should I remove it from this PR and make another one?

File: picard/config_upgrade.py [hunk 2]
```diff
@@ -191,7 +191,6 @@ def upgrade_to_v1_4_0_dev_4():
 def upgrade_to_v1_4_0_dev_5():
     """Using Picard.ini configuration file on all platforms"""
     # this is done in Config.__init__()
-    pass
```
Comment (mineo) [inline picard/config_upgrade.py]: I think a function declaration without any function body is actually invalid syntax.

File: picard/ui/ui_infodialog.py (hunk 1 above)
Comment (mineo) [inline picard/ui/ui_infodialog.py]: Hm, calling `build_ui` on master doesn't remove it, but calling it on this branch doesn't add it back in again, so it's not a big problem.

File: picard/config_upgrade.py (hunk 2 above)
Comment (samj1912) [inline picard/config_upgrade.py]: If it has a doc string then it is not.

**Pull request merged**